test/unit/news.spec.js

```diff
@@ -151,6 +149,11 @@ describe('News', () => {
       n.getTitle('en').should.equal('test');
     });
 
+    it('Should return a lang when a lang is supplied', () => {
+      const n = new News(languageTestData, { mdConfig, timeDate, translator });
+      n.getTitle('en').should.equal('/Lotus/Language/test/test');
+    });
```

Comment:
hmmmm this doesn't translate anything

Reply:
I mean the mock data for translations just returns the key, so in my head detecting '/Lotus/Languages' would mean that it would actually look for the matching key but if you'd like I can add a version of `realTestData` with the current discord lang that's out

Reply:
what i mean is that this is telling me your test is still returning exactly the same string, which it shouldn't
languageString should return the end of the i18n string or a translation

Reply:
but yeah,  you'd probably need to add the real one to realTestData and validate it

Reply:
okay I see the issue now

Reply:
Alright I'm sure I got it now